lib/PuppeteerSharp/Browser.cs

```diff
@@ -139,24 +147,30 @@ public class Browser : IDisposable
         /// Creates a new page
         /// </summary>
         /// <returns>Task which resolves to a new <see cref="Page"/> object</returns>
-        public async Task<Page> NewPageAsync()
-        {
-            string targetId = (await Connection.SendAsync("Target.createTarget", new Dictionary<string, object>
-            {
-                ["url"] = "about:blank"
-            }).ConfigureAwait(false)).targetId.ToString();
-
-            var target = TargetsMap[targetId];
-            await target.InitializedTask.ConfigureAwait(false);
-            return await target.PageAsync().ConfigureAwait(false);
-        }
+        public Task<Page> NewPageAsync() => _defaultContext.NewPageAsync();
 
         /// <summary>
         /// Returns An Array of all active targets
         /// </summary>
         /// <returns>An Array of all active targets</returns>
         public Target[] Targets() => TargetsMap.Values.Where(target => target.IsInitialized).ToArray();
 
+        public async Task<BrowserContext> CreateIncognitoBrowserContextAsync()
+        {
+            var response = await Connection.SendAsync<CreateBrowserContextResponse>("Target.createBrowserContext", new { });
+            var context = new BrowserContext(this, response.BrowserContextId);
+            _contexts[response.BrowserContextId] = context;
+            return context;
+        }
+
+        public BrowserContext[] BrowserContexts()
```

Comment:
I'm wondering whether this should be a method or a readonly property

Reply:
It can't be a readonly property because the method `CreateIncognitoBrowserContextAsync` adds a new context to the browser

Reply:
I mean something like this
```
public BrowserContext[] BrowserContexts
{
    get
	{
		var allContexts = new BrowserContext[_contexts.Count + 1];
        allContexts[0] = _defaultContext;
        _contexts.Values.CopyTo(allContexts, 0);
        return allContexts;
	}
}
```

Reply:
I made it a method to be like the `Targets` method